src/main/java/frc/robot/Robot.java

```diff
@@ -20,6 +20,12 @@
  */
 public class Robot extends TimedRobot {
   public static final boolean isCompetition = true;
+  public static final boolean batteryEnabled = true;
+  public static final boolean cameraEnabled = true;
+  public static final boolean everybotIntakeEnabled = true;
+  public static final boolean intakeEnabled = true;
+  public static final boolean simulationEnabled = true;
```

Comment:
What is the use case for disabling simulation?

Reply:
I don't know but it may come up at some point, it might just be nice to have.